src/applications/find-va-forms/components/Card.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import classnames from 'classnames';
+
+export default function Card({ heading, href, description }) {
+  const containerClassName = classnames(
+    'vads-u-display--block',
+    'vads-u-background-color--primary-alt-lightest',
+    'vads-u-padding--2',
+    'vads-u-height--full',
+    'vads-u-text-decoration--none',
+  );
+
+  return (
+    <a href={href} className={containerClassName}>
```

Comment:
Should these links include `aria-labels`? I know the info passed to them via props is pretty descriptive, but wasn't sure.

Reply:
Do we want `rel="noopener noreferrer"` here?

Reply:
Good call, I think a new prop for the `rel` would make sense. We generally only need to worry about that for pages outside of our domain though.

Reply:
@micahchiang Good call on the aria attribute! I'll add one.